room.h

```diff
@@ -174,6 +174,15 @@ namespace QMatrixClient
             /** Mark all messages in the room as read */
             void markAllMessagesAsRead();
 
+            void inviteCall(const QString& callId, const int& lifetime,
+                            const QString& sdp);
+            void callCandidates(const QString& callId,
+                                const QJsonArray& candidates);
+            void answerCall(const QString& callId, const int& lifetime,
+                            const QString& sdp);
+            void answerCall(const QString& callId, const QString& sdp);
+            void hangupCall(const QString& callId);
+
```

Comment:
Looking at those `callId`s in each call, I start to think that we might want to have a separate "1:1 call" class with its own lifecycle. The "call candidate" structure definition would land inside that class as well.